web/packages/teleterm/src/ui/Search/pickers/ActionPicker.tsx

```diff
@@ -523,6 +523,31 @@ function ClusterFilterItem(props: SearchResultItem<SearchResultCluster>) {
   );
 }
 
+function DisplayResultsItem(props: SearchResultItem<DisplayResults>) {
+  return (
+    <IconAndContent Icon={icons.Magnifier} iconColor="text.slightlyMuted">
+      <Text typography="body1">
+        {' '}
+        Display {props.searchResult.value ? 'search' : 'all'} results{' '}
+        {props.searchResult.value && (
```

Comment:
Weren't we supposed to show some other text if there's no filter and selecting this option will open a tab with search results but only for the current cluster?

https://gravitational.slack.com/archives/C03FJA391M3/p1699022780120479?thread_ts=1699019207.545239&cid=C03FJA391M3

Also, didn't the search bar use to have the cluster filter selected by default?

Reply:
That option with the cluster name on the right looks really nice. 👍

Reply:
Lack of a default filter is fine, I just didn't remember all decisions we made along the way and was surprised that something that used to be there is no longer present.